### PR TITLE
Branch name

### PR DESCRIPTION
**Pull request by Rohanksingh18**
I changed some variables. Because, during the first commit of these codes, which was not showing in git. therefore, I changed some variables values and committed again to review

### CODE REVIEW

**Review by admaskinfu — COMMENTED** [review]
I just noticed your 'Requirments.txt' file has upper case 'R'. Please change that to all lower case 'requirements.txt'. 
It works but it is convention. so you look more professional and experienced.

File: ecom_test/tests/backend_tests/products/test_list_products_with_filter.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+
+import pytest
+from ecom_test.src_file.api_helpers.ProductsAPIHelpers import ProductsAPIHelper
+from ecom_test.src_file.dao.products_dao import ProductsDAO
```
Comment (admaskinfu) [inline ecom_test/tests/backend_tests/products/test_list_products_with_filter.py:4]: I am not a fan of the name 'src_file'.
It is a direcotry but you are calling it 'file'.
Just call it 'src' that is very common and more professional.

File: ecom_test/tests/backend_tests/products/test_list_products_with_filter.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+from datetime import datetime, timedelta
+
+
+@pytest.mark.regression
+class TestListProductsWithFilter(object):
+
+    # create a method to get the list of products with filter for specific period of time from now
+
+    @pytest.mark.tcid35
```
Comment (admaskinfu) [inline ecom_test/tests/backend_tests/products/test_list_products_with_filter.py:13]: I am assuming "tcid35" is your own. So it is difficult for me to match it with mine. Incase I want to compare the code to mine.
I think your tcid35 is same as mine tcid51. Can you may be add another mark (may be @pytest.mark.ssqaid51. We can discuss this further if you like.

File: ecom_test/tests/backend_tests/products/test_list_products_with_filter.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    def test_list_products_with_filter_after(self):
+        # Create a payload data request to be sent
+        x_days_from_now = 60  # variable for requesting data for last 30 days form now
```
Comment (admaskinfu) [inline ecom_test/tests/backend_tests/products/test_list_products_with_filter.py:16]: You are doing 60 but the comment says 30

File: ecom_test/tests/backend_tests/products/test_list_products_with_filter.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+        rs_api = ProductsAPIHelper().call_list_products(pyload)
+        assert rs_api, f" Empty response for 'list products with filter'"
+
+        # get the data from database
+
+        db_products = ProductsDAO().get_products_created_after_given_date(after_date)
+
+        # verify the responses to match DB
+
```
Comment (admaskinfu) [inline ecom_test/tests/backend_tests/products/test_list_products_with_filter.py:33]: not a fan of the space after the comment. but good job adding comments.

File: ecom_test/tests/backend_tests/products/test_update_regular_price_updates_price.py [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+import pytest
+from ecom_test.src_file.api_helpers.ProductsAPIHelpers import ProductsAPIHelper
+from ecom_test.src_file.utilities.generic_utilities import generate_random_string
+import random
+
+pytestmark = [pytest.mark.products, pytest.mark.regression]
+
+
+@pytest.mark.tcid36
+def test_update_regular_price_updates_price():
```
Comment (admaskinfu) [inline ecom_test/tests/backend_tests/products/test_update_regular_price_updates_price.py:11]: Add a 'doc string' to the function. Explaining what the function is testing.
May be do research if doc string is common/expected in tests.

File: ecom_test/tests/backend_tests/products/test_list_products_with_filter.py (hunk 1 above)
Comment (Rohanksingh18) [reply ecom_test/tests/backend_tests/products/test_list_products_with_filter.py:4]: will take care of for the next commit

File: ecom_test/tests/backend_tests/products/test_list_products_with_filter.py (hunk 3 above)
Comment (Rohanksingh18) [reply ecom_test/tests/backend_tests/products/test_list_products_with_filter.py:16]: changed

File: ecom_test/tests/backend_tests/products/test_list_products_with_filter.py (hunk 4 above)
Comment (Rohanksingh18) [reply ecom_test/tests/backend_tests/products/test_list_products_with_filter.py:33]: extra spaces removed

File: ecom_test/tests/backend_tests/products/test_update_regular_price_updates_price.py (hunk 5 above)
Comment (Rohanksingh18) [reply ecom_test/tests/backend_tests/products/test_update_regular_price_updates_price.py:11]: WIP

File: ecom_test/tests/backend_tests/products/test_list_products_with_filter.py (hunk 2 above)
Comment (Rohanksingh18) [reply ecom_test/tests/backend_tests/products/test_list_products_with_filter.py:13]: it's different from yours

**Pull request merged**